### DISCUSSION TITLE
.column-page in typst

### DISCUSSION THREAD

**Discussion by fredguth** [quarto-dev/quarto-cli#11568] [Q&A]
### Description

I am using a `manuscript` project and I want to create a section with the `.column-page` layout.
```md
:::{.column-page}

 code block

:::
```
It works in the website, but not in the typst pdf.  Is there a way to change the layout in typst?

**Comment by fredguth** [comment]
I found out a work around changing all raw blocks in the typst templat:

```{=typst}
#show raw.where(block: true): block.with(
    fill: rgb("#f6f6f0d9"), 
    width: 100%+6cm,      <---- HERE !
    inset: 8pt, 
    radius: 2pt
  )
  ``` 
  
  But not exactly what I wanted.

**Reply by gordonwoodhull** [reply]
Thanks for the workaround! Indeed we have a ways to go on feature parity for layout features in Typst.

Your [tufte-inspired](https://github.com/fredguth/tufte-inspired) format extension fills many of these gaps btw! Eventually we would like to make it more consistent with the HTML features and bring it in, if you're interested in helping with that.
